contracts/interfaces/ILimitOrder.sol

```diff
@@ -7,10 +7,30 @@ import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
 import "./IStrategyBase.sol";
 import "../utils/LimitOrderLibEIP712.sol";
 
+/// @title ILimitOrder Interface
+/// @author imToken Labs
 interface ILimitOrder is IStrategyBase {
+    /// @notice Emitted when coordinator address is updated
+    /// @param newCoordinator The address of the new coordinator
     event UpgradeCoordinator(address newCoordinator);
+
+    /// @notice Emitted when fee factors is updated
+    /// @param makerFeeFactor The new fee factor for maker
+    /// @param takerFeeFactor The new fee factor for taker
+    /// @param profitFeeFactor The new fee factor for relayer profit
     event FactorsUpdated(uint16 makerFeeFactor, uint16 takerFeeFactor, uint16 profitFeeFactor);
+
+    /// @notice Emitted when fee factors is updated
```

Comment:
```suggestion
    /// @notice Emitted when fee collector address is updated
```

Reply:
Adjusted in 7a91187197c37a8285b47ee4f3227bad434ebccb.